apache2/apache2_util.c

```diff
@@ -187,6 +192,115 @@ char *get_env_var(request_rec *r, char *name) {
     return result;
 }
 
+#ifdef WAF_JSON_LOGGING_ENABLE
+/**
+ * Retrieve waf log field.
+ */
+void get_field_value(const char* from, const char* to, const char* text, char* output) {
+    char* first = strstr(text, from);
+    int first_index = first - text;
+
+    if (first != NULL ) {
+        if ((first_index > 0 && (first-1)[0] != "\\") || (first_index == 0)) {
```

Comment:
be 100% sure that it works ] in msg, or \", or \\

Reply:
yes, checked it

Reply:
@azhao155 could you make this function `static`